cmd/deployment-check/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.13 AS builder
+RUN apt-get upgrade -y && apt-get install git -y
+ADD . /go/src/app
+WORKDIR /go/src/app
+RUN go get -v
+RUN go build -o app
+RUN mkdir /app
+RUN mv app /app/app
+
+FROM golang:1.13
```

Comment:
Move to an alpine image here.  You may need to set `GOOS=linux` in the builder and will need to `apk update && apk add ca-certificates`.